src/locales/fr/WCAG.json

```
@@ -0,0 +1,729 @@
+            "TITLE": "À la saisie",
+            "DESCRIPTION": "Le changement de paramètre d’un composant d’interface utilisateur ne doit pas provoquer de changement de contexte à moins que l’utilisateur n’ait été avisé de ce comportement avant d’utiliser le composant.",
+            "DETAILS": {}
+        },
+        "3.2.3": {
+            "TITLE": "Navigation cohérente",
+            "DESCRIPTION": "Dans un ensemble de pages, les mécanismes de navigation qui se répètent sur plusieurs pages Web se présentent dans le même ordre relatif chaque fois qu’ils sont répétés, à moins qu’un changement soit provoqué par l’utilisateur.",
+            "DETAILS": {}
+        },
+        "3.2.4": {
+            "TITLE": "Identification cohérente",
+            "DESCRIPTION": "Dans un ensemble de pages Web, les composants qui ont la même fonctionnalité sont identifiés de la même façon.",
+            "DETAILS": {}
+        },
+        "3.2.5": {
+            "TITLE": "Changement à la demande",
+            "DESCRIPTION": "Un changement de contexte est provoqué uniquement sur demande de l’utilisateur ou un mécanisme est disponible pour désactiver un tel changement.",
+            "DETAILS": {}
+        },
+        "3.3.1": {
+            "TITLE": " Identification des erreurs",
+            "DESCRIPTION": "Si une erreur de saisie est détectée automatiquement, l’élément en erreur est identifié et l’erreur est décrite à l’utilisateur sous forme de texte.",
+            "DETAILS": {}
+        },
+        "3.3.2": {
+            "TITLE": "Étiquettes ou instructions",
+            "DESCRIPTION": "Des étiquettes sont présentées ou des instructions sont fournies quand un contenu requiert une saisie utilisateur.",
+            "DETAILS": {}
+        },
+        "3.3.3": {
+            "TITLE": "Suggestion après une erreur",
+            "DESCRIPTION": "Si une erreur de saisie est automatiquement détectée et que des suggestions de corrections sont connues, ces suggestions sont alors proposées à l’utilisateur à moins que cela puisse compromettre la sécurité ou la finalité du contenu.",
+            "DETAILS": {}
+        },
+        "3.3.4": {
+            "TITLE": "Prévention des erreurs (juridiques, financières, de données)",
+            "DESCRIPTION": "Pour les pages Web qui entraînent des engagements juridiques ou des transactions financières de la part de l’utilisateur, qui modifient ou effacent des données contrôlables par l’utilisateur dans des systèmes de stockages de données, qui enregistrent les réponses de l’utilisateur à un test ou un examen, au moins l’une des conditions suivantes est vraie :",
+            "DETAILS": {
+                "1": {
+                    "DESCRIPTION": "Les actions d’envoi sont réversibles.",
+                    "TITLE": "Réversible"
+                },
+                "2": {
+                    "DESCRIPTION": "Les erreurs de saisie de l’utilisateur sont vérifiées et la possibilité est donnée à l’utilisateur de les corriger.",
```

```suggestion
                    "DESCRIPTION": "les données saisies par l’utilisateur sont vérifiées au niveau des erreurs de saisie et la possibilité est donnée à l’utilisateur de les corriger.",
```

See https://www.w3.org/Translations/WCAG21-fr/#error-prevention-legal-financial-data
Please use the exact wording of the Authorized Translation

---

Thanks for the missing capital letter: I've updated my suggestion.

Concerning the wording, see https://github.com/w3c/wai-wcag-em-report-tool/pull/127#discussion_r1402411577